.github/workflows/ci.yml

```diff
@@ -42,8 +41,10 @@ jobs:
     strategy:
       matrix:
         include:
-          - feature: 
           - feature: default
+        target:
+          - wasm32-unknown-unknown
+          - wasm32-wasi
```

Comment:
I think only adding this wouldn't really make cargo build for them, should we also add `--target "${{ matrix.target }}"` in the following commands?

Also I think we could separate the build check for wasm as another job (just like halo2 [test](https://github.com/privacy-scaling-explorations/halo2/blob/main/.github/workflows/ci.yml#L11) and [build](https://github.com/privacy-scaling-explorations/halo2/blob/main/.github/workflows/ci.yml#L35) is separated), that might make the CI faster?

Reply:
The idea is that the `test` step uses the catched artifacts from the `build` step in the CI. I agree anyways we can have a sepparated job for the WASM-compat which will anyways not be re-used.

But I'd not split all the build from all the test in general due to the cache optimizations done for now with `nextest`.

As for the `target`. I completelly missed it! Thx!

Reply:
Fixed in [c5acdaf](https://github.com/privacy-scaling-explorations/halo2curves/pull/122/commits/c5acdafea008745b513f05dd169f72da71c28a22)